src/main/kotlin/org/malachite/estella/services/OrganizationService.kt

```diff
@@ -23,4 +23,38 @@ class OrganizationService(@Autowired private val organizationRepository: Organiz
     }
 
     fun deleteOrganization(id: UUID) = organizationRepository.deleteById(id)
+
+    fun verifyOrganization(uuid: String): Organization =
+        changeOrganizationVerification(uuid, true)
+
+
+    fun deverifyOrganization(uuid: String): Organization =
+        changeOrganizationVerification(uuid, false)
+
+    fun changeOrganizationVerification(uuid: String, verified: Boolean): Organization {
+        val organization = addOrganization(
+            getOrganization(UUID.fromString(uuid))
+                .copy(verified = verified)
+        )
+        //send mail
```

Comment:
Nie wiem czy najlepszym pomysłem jest trzymanie templateów maili w serwisie

Reply:
Ja bym w ogóle postulował, żeby one gdzieś na zewnątrz były, ale to może być kiepskie performance'owo

Reply:
Na zewnątrz że w bazie?

Reply:
Na zewnątrz, że w jakimś osobnym serwisie całkowicie XDDD w sensie osobny serwis do templatek, gdzie będziemy też mogli trzymać templatki organizacji, ale może to być przesada na teraz i jako propozycja rozwoju projektu

Reply:
Znaczy wydaje mi się to kiepskim rozwiązaniem bo czym byłby taki serwis w ogóle listą templateów maili? i byśmy fetchowali ten template i wklejali na froncie ten pobrany template?

Reply:
Yup exactly XDD